src/AddIdea.tsx

```diff
@@ -40,9 +40,11 @@ import {
 } from '@components/ui/popover';
 import { useToast } from '@components/ui/use-toast';
 import { Input } from '@components/ui/input';
-import { Idea } from '@utils/types'
+import { Idea, ICategory } from '@utils/types';
```

Comment:
I find the "ICategory" not that much explicit, is it a convention ? (probably for interface i think)

Reply:
Exactly, it's to differentiate between an interface and a type.
I don't know if there's a specific convention for this, but I personally find it clearer.